.github/workflows/e2e-matrix.yaml

```diff
@@ -45,6 +45,7 @@ jobs:
       suite: ${{ matrix.suite }}
       git_ref: ${{ needs.resolve-git-ref.outputs.GIT_REF }}
       event_name: ${{ github.event_name }}
+      region: ${{ vars.AWS_REGION }}
```

Comment:
If you pass `vars.AWS_REGION` to this, then the top-level caller can't set this. My assumption is that you are trying to enable conformance tests to call this so I think you want to take this value as an input and then offer options with a sensible default to the caller.